packages/runtime/datastore/src/remoteChannelContext.ts

```diff
@@ -80,6 +70,46 @@ export class RemoteChannelContext implements IChannelContext {
 			extraBlobs,
 		);
 
+		this.channelP = new LazyPromise<IChannel>(async () => {
+			const { attributes, factory } = await loadChannelFactoryAndAttributes(
+				dataStoreContext,
+				this.services,
+				this.id,
+				registry,
+				attachMessageType,
+			);
+
+			const channel = await loadChannel(
```

Comment:
any reason we don't want analogous data processing error wrapping in this codepath? do we have some more outer-level wrapping for remote channels?

behavior here looks like it exactly matches previous behavior, so i'm fine not making a change here. just wondering.

Reply:
i'm not going to add it for now. i'm pretty sure it will get wrapped at a higher level, but regardless i've trying to keep this refactor fairly mechanical, so avoiding changing things not relevant to the refactor, which is always fuzzy line to walk